target/linux/ipq40xx/image/generic.mk

```diff
@@ -980,6 +981,35 @@ define Device/tel_x1pro
 endef
 TARGET_DEVICES += tel_x1pro
 
+define Device/tplink_deco-m5
+        $(call Device/FitImage)
+        DEVICE_VENDOR := TP-Link
+        DEVICE_MODEL := Deco-M5
+        SOC := qcom-ipq4019
+        DEVICE_PACKAGES := ipq-wifi-tplink_deco-m5
+        TPLINK_BOARD_ID := DECO-M5
+        IMAGE_SIZE := 16640k
+        IMAGES += factory.bin
+        IMAGE/factory.bin := append-rootfs | tplink-safeloader factory
+        IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | append-metadata
+endef
+TARGET_DEVICES += tplink_deco-m5
+
+define Device/tplink_deco-m5-v2
```

Comment:
Can you please make a common profile and reuse it for both models instead of duplicating